docs/Data_Portal/Users_Guide/quick_start.md

```diff
@@ -18,10 +18,10 @@ On the upper-left is the GDC Data Portal logo, which links to the home page of t
 On the right are the following features:
 * **Video Guides**: links to videos demonstrating the various features of the Data Portal
 * **Send Feedback**: provides information on how to send feedback to our team
-* **Browse Annotations**: links to the Annotations Browser, where the user can view and search for annotations that may be of use when analyzing GDC data
+* **Browse Annotations**: links to the Annotations Browser, where the user can view and search for [annotations](/Encyclopedia/pages/Annotations/) that may be of use when analyzing GDC data
```

Comment:
I think this might need to be (/Encyclopedia/pages/Annotations.md)

Reply:
The *.md format seems to be what works based on what I see in the docs now.  I think without the *.md, the full URL will work (though it will be treated as an "external link").  

I'm not sure whether the "no .md" relative URLs will work when the site is built, they might and I'm curious, but it's probably not worth testing right now.  So let's move forward with the .md suffixes and we'll test this later.